src/modules/common/components/Caption.styles.js

```diff
@@ -1,6 +1,6 @@
-import styled from 'styled-components'
+// @flow
 
-// fixme: H1 is connected to redux state -> not a plain old component
```

Comment:
It is not connected to redux but to an other context. So this is still not a plain component.

Reply:
Then we probably need to adjust our definition of component/container, otherwise everythings a container (almost)

Reply:
Yep, will write one after thinking about them in the react native app. It actually is pretty clear an makes sense. A lot of containers will be pretty simple and just contain one or two lines.